cdap-ui/app/features/dashboard/routes.js

```diff
@@ -1,38 +1,77 @@
 angular.module(PKG.name+'.feature.dashboard')
   .config(function ($stateProvider, $urlRouterProvider, MYAUTH_ROLE) {
 
+    var path = '/assets/features/operation28/';
```

Comment:
Yes. The the controllers and templates are still under operations28 directory. We are just re-using stuff from that directory instead of moving them to dashboard.